### DISCUSSION TITLE
可以考虑使用sqlite without cgo的驱动

### DISCUSSION THREAD

**Discussion by YiuTerran** [loggie-io/loggie#607] [Ideas]
即[这个](https://gitlab.com/cznic/sqlite)库，不过有[测试](https://datastation.multiprocess.io/blog/2022-05-12-sqlite-in-go-with-and-without-cgo.html)表明该库的性能比CGO版本的稍差。

**Comment by ethfoo** [comment]
可以考虑一下，现在有非CGO有badger的，但是资源占用（memory，磁盘）都是比较大，后面调整过badger的参数，仍然不是太理想。
